onegov/swissvotes/tests/test_models.py

```diff
@@ -1358,3 +1002,36 @@ def test_column_mapper():
     assert list(mapper.items())[302] == (
         '!recommendations!sodk', 'p-sodk', 'INTEGER', True, None, None
     )
+
+
+def test_recommendation_order():
+    recommendations = SwissVote.codes('recommendation')
+    assert list(recommendations.keys()) == [1, 2, 4, 5, 3, 66, 9999, None]
+
+
+def test_recommendations_parties(sample_vote):
+    grouped = sample_vote.recommendations_parties
+    codes = SwissVote.codes('recommendation')
+    # Remove entries in codes for unknown and actor no longer exists
+    del codes[9999]
+    # del codes[None]
+    # print(grouped.keys())
+    # print(codes.values())
+    assert list(grouped.keys()) == list(codes.values())
+
+
+def test_sorted_actors_list(sample_vote):
+    sorted_actors = sample_vote.sorted_actors_list
+    print(sorted_actors)
+    assert sorted_actors
+    for i in range(len(sorted_actors) - 2):
+        actor = sorted_actors[i]
+        next_actor = sorted_actors[i + 1]
+        actor_ = actor
```

Comment:
Here you could reuse the `normalize_actor_shortcut(shortcut)` function suggested above.